smart-contract-example/README.md

```diff
@@ -0,0 +1,90 @@
+## Smart Contract Workflow Example
+
+This example demonstrates how Kurtosis can be used for smart contract dapp development 
+by using the [`eth-network-package`](https://github.com/kurtosis-tech/eth-network-package) as a local Ethereum network. 
+The `eth-network package` can be used as a low overhead, configurable, and composable alternative to services like
+`hardhat-network`, `ganache` and `anvil`.
+
+### Setup
+
+This folder contains a typical setup for a smart contract developer looking to develop a dapp using the 
+[Hardhat](https://hardhat.org/) framework. Let's explore it's contents.
+- `contracts/` contains a few simple smart contracts for a Blackjack dapp 
+- `scripts/` contains a script to deploy a token contract to our local Ethereum network
+- `test/` contains a simple test for our token contract
+- `hardhat.config.ts` configures our Hardhat setup. 
+It allows us to configure Hardhat to use a local Ethereum network created by the `eth-network-package`.
+
+### Running the Example
+
+This assumes you have the following services installed:
```

Comment:
Probably should add Docker here too because you need Docker to use Kurtosis. Specifically installing Docker and starting up the Docker Daemon. I'd recommend linking to our docs about this here https://docs.kurtosis.com/install#i-install--start-docker